p/themes/Swage/swage.css.map

```diff
@@ -1 +1 @@
AAA,CACA,UAAA,CAEA,uBACC,oBAAA,CACA,iBAAA,CACA,UAAA,CAEA,6BACC,cAAA,CACA,gBAAA,CAMD,+DACC,oBAAA,CAKH,MACC,kBAAA,CACA,kBAhhCa,CAkhCb,aAGC,aAAA,CACA,SAAA,CACA,4BAAA,CAGD,2BAEC,eAphCY,CAshCZ,2FACC,eAvhCW,CA2hCb,6BACC,kBA9hCa,CAgiCb,+CACC,kBAjiCY,CAqiCd,6BACC,kBAriCa,CAuiCb,+CACC,kBAxiCY,CA6iCb,iBACC,aAAA,CACA,eAAA,CAIF,cACC,eAAA,CACA,iBAAA,CAGD,wBACC,WAAA,CAGD,YACC,aApkCY,CAqkCZ,cAAA,CAIF,aACC,eAAA,CACA,cAAA,CAEA,oBACC,eAAA,CAIF,cACC,cAAA,CACA,kBAAA,CACA,WAAA,CACA,aAAA,CACA,aAAA,CACA,WAAA,CACA,iBAAA,CACA,gBAAA,CACA,qBAAA,CACA,cAAA,CACA,WAAA,CACA,MAAA,CACA,QAAA,CAIA,qCAEC,aArmCY,CAwmCb,mBACC,kBArmCW,CAumCX,iCACC,kBAxmCU,CA4mCZ,kBACC,kBAAA,CAEA,gCACC,kBAAA,CAIF,gCACC,eAAA,CAGD,sBACC,YAAA,CAIF,eACC,iBAAA,CACA,oBAAA,CACA,kBAAA,CAEA,qBACC,kBAroCY,CAsoCZ,aAxoCY,CA4oCd,aACC,QAAA,CACA,iBAAA,CACA,eAAA,CACA,kBAAA,CAGA,kBAjpCa,CAopCd,MACC,kBAAA,CAEA,2BAGC,WAAA,CAIA,8BAEC,+BAAA,CAIF,uBACC,cAAA,CAEA,6BACC,eAAA,CACA,kBAAA,CACA,sBAAA,CAEA,yCACC,WAAA,CAMJ,SACC,WAAA,CAGD,OACC,WAAA,CAEA,iBACC,iBAAA,CAMD,qCACC,QAAA,CAGD,qCACC,kBAvsCW,CAwsCX,WAAA,CACA,iBAAA,CAGC,qEACC,WAAA,CACA,UAAA,CACA,iBAAA,CACA,OAAA,CACA,kBAAA,CACA,QAAA,CAEA,iFACC,oBAAA,CAOL,SACC,aA9tCa,CA+tCb,aAAA,CAGD,SACC,eAAA,CACA,wBAAA,CAGD,KACC,cAAA,CACA,kBAzuCa,CA0uCb,eAAA,CACA,aAAA,CAEA,WACC,iBAAA,CACA,gBAAA,CAGD,iBACC,kBAAA,CACA,aApvCY,CAuvCb,mBACC,kBArvCY,CAsvCZ,aAzvCY,CA4vCb,kBACC,kBAAA,CACA,aA9vCY,CAiwCb,iBACC,kBAnwCW,CAowCX,aAnwCY,CAuwCd,yBAGG,mDACC,YAAA,CAKH,wBACC,gBAAA,CACA,eAAA,CAGD,UACC,iBAAA,CAGD,aACC,eAAA,CACA,UAAA,CAGD,QACC,aAAA,CAEA,cACC,SAAA,CAGD,wBACC,SAAA,CACA,cAAA,CACA,UAAA,CAEA,8BACC,cAAA,CAKH,oBACC,YAAA,CAGD,8CAEC,UAAA,CACA,iBAAA,CAGD,OACC,SAAA,CACA,OAAA,CACA,6BAAA,CAEA,qBACC,kBA9zCW,CA+zCX,aAAA,CACA,WAAA,CACA,gBAAA,CACA,iBAAA,CACA,kBAAA,CAEA,2BACC,wBAAA,CAGD,2BACC,oBAAA,CAIF,mCACC,YAAA,CACA,YAAA,CACA,gBAAA,CACA,iBAAA,CAGD,cACC,SAAA,CACA,YAAA,CAKD,sBACC,kBA71CW,CA81CX,aAAA,CACA,WAAA,CACA,gBAAA,CACA,iBAAA,CACA,kBAAA,CAEA,4BACC,wBAAA,CAGD,4BACC,oBAAA,CAKH,UACC,WAAA,CAEA,eACC,QAAA,CAGD,iBACC,QAAA,CAEA,sBACC,QAAA,CAIF,uBACC,OAAA,CACA,gBAAA,CACA,wBAAA,CACA,iBAAA,CAEA,4BACC,aAAA,CAGD,6BACC,UAAA,CAKH,2CAEC,kBA/4CY,CAg5CZ,aAAA,CACA,WAAA,CACA,gBAAA,CACA,iBAAA,CACA,kBAAA,CAEA,uDACC,wBAAA,CAIF,oGAEC,YAAA,CAGD,sBACC,QAAA,CACA,SAAA,CAGD,WACC,gBAAA,CAGD,cACC,UAAA,CAEA,sBACC,wBAAA,CACA,aAAA,CACA,MAAA,CAEA,4BACC,UAAA,CAGD,4BACC,YAAA,CAKH,aACC,qBAAA,CAGD,MACC,iBAAA,CACA,kBAAA,CAGD,WACC,cAAA,CAGD,mBACC,iBAAA,CACA,KAAA,CAGD,uFAKC,iBAAA,CAGD,oBACC,UAAA,CACA,KAAA,CAGD,2BAEC,OAAA,CAGD,YACC,KAAA,CAGD,uDAGC,QAAA,CAGD,qBACC,QAAA,CAGD,mBACC,UAAA,CAAA,CAIF,yBACC,iBACC,QAAA,CAAA,CAIF,yBACC,gBACC,YAAA,CAAA,CAIF,eACC,YAAA,CAGD,6CACC,kBAAA,CAGD,uCACC,kBAAA,CACA,gBAAA,CAIA,iBACC,aA5gDY,CA8gDZ,wBACC,kBA/gDW,CAghDX,aAnhDW,CAqhDX,0BACC,aAthDU,CA0hDZ,mBACC,aAxhDW,CA4hDb,iBACC,aAAA,CAEA,wBACC,kBAAA,CACA,aApiDW,CAsiDX,0BACC,aAviDU,CA2iDZ,mBACC,aAAA,CAKH,qBACC,gBAAA,CACA,kBAnjDa,CAojDb,aAljDa,CAmjDb,WAAA,CAEA,6BACC,eAAA,CACA,aAAA,CACA,aAAA,CAKD,mCACC,MAAA,CACA,UAAA,CAKD,oCACC,OAAA,CACA,SAAA,CAKD,cACC,kBAAA","file":"swage.css"}
```

Comment:
I do not have knowledge about `.map`. I am not sure if it make sense to remove `sourceRoot`. Could you please support us with more information?

Reply:
I use this extension: https://github.com/glenn2223/vscode-live-sass-compiler
I haven't dealt with the map files that much either, but so far it has always worked very well, with no errors in the browser console and the w3 tests haven't given any error messages either.